portal-impl/src/com/liferay/portlet/exportimport/service/impl/ExportImportLocalServiceImpl.java

```diff
@@ -1,779 +0,0 @@
-/**
- * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
- * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
- */
-
-package com.liferay.portlet.exportimport.service.impl;
-
```

Comment:
Hey @MrJohn1911, update the commit message to say that you moved the Service and ServiceImpl to the Manager class. Thx